bakerydemo/static/css/main.css

```diff
@@ -14,81 +14,182 @@
 /* 9. Miscellaneous/ Helper styles ------------------------------------------ */
 
 /* Global styles */
+/* CSS Variables */
+:root {
+  --black: #000000;
+  --dark: #333333;
+  --grey: #6e6e6e;
+  --border-grey: #E1DCD3;
+  --white: #ffffff;
+  --cream: #f5f3e9;
+  --light-brown: #87744f;
+  --mid-brown: #825600;
+  --dark-brown: #553801;
+  --orange: #c55302;
+  --dark-orange: #833701;
+  --font--primary: "Marcellus", sans-serif;
+  --font--secondary: "Open Sans", sans-serif;
+}
+
+/* Utilities */
+
+.u-sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
+  white-space: nowrap;
+  border: 0;
+}
+
+/* Body */
+
 body {
+  font-family: var(--font--secondary);
+  font-size: 16px;
+  line-height: 24px;
   padding-top: 0;
   padding-bottom: 0;
-  font-size: 19px;
-  font-family: 'Alegreya', serif;
-  background: rgb(255,255,255);
-  background: linear-gradient(135deg,  rgba(255,255,255,1) 0%,rgba(255,255,255,1) 50%,rgba(233,228,221,1) 100%);
+  background: var(--white);
+  color: var(--black);
   min-height: 100vh;
 }
 
 @media (min-width: 768px) {
   body {
-    font-size: 22px;
+    font-size: 18px;
+    line-height: 28px;
   }
 }
 
-h1 {
-  font-weight: 300;
-  font-family: 'Lato', sans-serif;
-  position: relative;
+/* Headings and Paragraphs */
+
+h1,
+h2,
+h3 {
+  font-family: var(--font--primary);
+  font-weight: 400;
+  color: var(--dark);
 }
 
-.hero h1 {
-  color: white;
+h1 {
+  font-size: 38px;
+  line-height: 44px;
 }
 
 h2 {
-  font-weight: 900;
-  font-family: 'Lato', sans-serif;
-  color: #63210d;
+  font-size: 32px;
+  line-height: 40px;
 }
 
 h3 {
-  font-weight: 900;
-  font-family: 'Lato', sans-serif;
-  color: #63210d;
+  font-size: 26px;
+  line-height: 30px;
 }
 
 h4 {
+  font-family: var(--font--secondary);
+  font-style: normal;
   font-weight: 600;
-  font-family: 'Lato', sans-serif;
-  color: #63210d;
-  font-size: 22px;
+  font-size: 20px;
+  line-height: 27px;
+  color: var(--dark);
 }
 
 p {
-  margin: 0 0 30px;
-  color: #444;
+  font-family: var(--font--secondary);
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: var(--black);
+}
+
+@media (min-width: 768px) {
+  body {
+    font-size: 18px;
+    line-height: 28px;
+  }
+
+  h1 {
+    font-size: 60px;
+    line-height: 64px;
+  }
+
+  h2 {
+    font-size: 46px;
+    line-height: 50px;
+  }
+
+  h3 {
+    font-size: 32px;
+    line-height: 42px;
+  }
+
+  h4 {
+    font-size: 24px;
+    line-height: 33px;
+  }
+
+  p {
+    font-size: 18px;
+    line-height: 28px;
+    margin: 0 0 30px;
+  }
+}
+
+ol,
+ul {
+  margin-bottom: 30px;
+}
+
+.hero h1 {
+  color: white;
```

Comment:
Be nice if we were consistent with our variables usage even if it’s for things like "white".